[2024-03-17T00:27:39.179+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Airflow_Lab2_Flask.start_Flask_API manual__2024-03-17T00:27:37.580508+00:00 [queued]>
[2024-03-17T00:27:39.263+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Airflow_Lab2_Flask.start_Flask_API manual__2024-03-17T00:27:37.580508+00:00 [queued]>
[2024-03-17T00:27:39.272+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-17T00:27:39.276+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-03-17T00:27:39.366+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-17T00:27:39.477+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): start_Flask_API> on 2024-03-17 00:27:37.580508+00:00
[2024-03-17T00:27:39.484+0000] {standard_task_runner.py:55} INFO - Started process 1316 to run task
[2024-03-17T00:27:39.498+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab2_Flask', 'start_Flask_API', 'manual__2024-03-17T00:27:37.580508+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/Flask_API.py', '--cfg-path', '/tmp/tmpqwgv83tw']
[2024-03-17T00:27:39.501+0000] {standard_task_runner.py:83} INFO - Job 25: Subtask start_Flask_API
[2024-03-17T00:27:39.661+0000] {task_command.py:388} INFO - Running <TaskInstance: Airflow_Lab2_Flask.start_Flask_API manual__2024-03-17T00:27:37.580508+00:00 [running]> on host d8b0252a8521
[2024-03-17T00:27:40.207+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Airflow_Lab2_Flask
AIRFLOW_CTX_TASK_ID=start_Flask_API
AIRFLOW_CTX_EXECUTION_DATE=2024-03-17T00:27:37.580508+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-17T00:27:37.580508+00:00
[2024-03-17T00:27:40.232+0000] {logging_mixin.py:137} INFO -  * Serving Flask app 'unusual_prefix_fb1fa9258fa1ad786b4829a3672e98fa2e7542a0_Flask_API'
[2024-03-17T00:27:40.233+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2024-03-17T00:27:40.234+0000] {logging_mixin.py:137} WARNING - Address already in use
[2024-03-17T00:27:40.234+0000] {logging_mixin.py:137} WARNING - Port 5001 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2024-03-17T00:27:40.273+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-03-17T00:27:40.296+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
